### PR TITLE
Example Playbooks

### PR DESCRIPTION
**Pull request by rthatcher**

### CODE REVIEW

File: docs/source/tasks/enable-fabricv2-capability.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
+..
+.. SPDX-License-Identifier: Apache-2.0
+..
+
+Enable Hyperledger Fabric V2 Capability
+=======================================
+
+Fabric V2 capability is distinct from Favric V2.x binaries, and the most common scenario will be where the binaries on a network have been upgraded to v2.x, leaving the Capability of the channels at V1.x.  This playbook example can be used to upgrade firstly the system channel, then an array of named application channels.
```
Comment (sstone1) [inline docs/source/tasks/enable-fabricv2-capability.rst:8]: Typo for `Favric` - also, can we make the capitalisation consistent with `V` and `v` for versions?

File: docs/source/tasks/haca.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+The remaining values must always be set:
+
+* Set ``ha_ca_name`` to the name of the new certificate authority, for example ``HAOrg1 CA``.
+* Set ``ca_admin_identity`` to the name of the CA administrator enroll ID.
+* Set ``ca_admin_pass`` to the CA administrator enroll secret.
+* Set ``ca_admin_type`` to ``client`` if you are **not** using NodeOU support or ``admin`` if you **are** using NodeOU support.
+* Set ``db_datasource`` to the connection details for your PostgresSQL database, for example:
+ ``host=mypostgressql.example.com port=999 user=myUsername password=myPassword dbname=mydb sslmode=verify-full``
+* Set ``da_certfile1`` to the Base64 encoded value of the certificate for the PostgresSQL database.
```
Comment (sstone1) [inline docs/source/tasks/haca.rst:68]: Should this be `db_certfile1`?

**Review by sstone1 — APPROVED** [review]
Couple of minor typos but otherwise looks good 👍

**Review by sstone1 — CHANGES_REQUESTED** [review]
Need fixes for the typos but happy to merge!

File: docs/source/tasks/enable-fabricv2-capability.rst (hunk 1 above)
Comment (rthatcher) [reply docs/source/tasks/enable-fabricv2-capability.rst:8]: Typos corrected.

File: docs/source/tasks/haca.rst (hunk 2 above)
Comment (rthatcher) [reply docs/source/tasks/haca.rst:68]: Correction made

**Comment by sstone1** [comment]
@azp run

**Pull request merged**